Presentation.Web/Controllers/API/V2/Internal/Organizations/OrganizationGridInternalV2Controller.cs

```diff
@@ -82,6 +87,26 @@ public IHttpActionResult GetGridConfiguration([NonEmptyGuid] Guid organizationUu
                 .Match(Ok, FromOperationError);
         }
 
+        [HttpGet]
+        [Route("permissions")]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public IHttpActionResult GetGridPermissions([NonEmptyGuid] Guid organizationUuid)
+        {
+            if (!ModelState.IsValid) return BadRequest(ModelState);
+            return MapUuidToId(organizationUuid)
+                .Select(orgId => new OrganizationGridPermissionsResponseDTO
+                    { HasConfigModificationPermissions = HasModifyPermissions(orgId) })
+                .Match(Ok, FromOperationError);
+        }
+
+        private bool HasModifyPermissions(int orgId)
+        {
```

Comment:
Don't forget to create a simple Unit test for the newly created method in the service